packages/plugin-vue/README.md

```diff
@@ -83,7 +83,7 @@ const vueI18nPlugin = {
       return
     }
     if (/\.ya?ml$/.test(id)) {
-      code = JSON.stringify(require('js-yaml').safeLoad(code.trim()))
+      code = JSON.stringify(require('js-yaml').load(code.trim()))
```

Comment:
This may be a breaking change

Reply:
according to changelog its not not breaking as it defaults to being safe

Reply:
Yeah but as I understood, this are docs. I thought it represents examples, so others that uses this needs to also apply this change. I didn't read much further ^^'

Reply:
Okay, you may be right, cause it's just a `devDependency`. So plugin maintainers would need to use it on there own with their own version of `js-yaml`. :+1: